tianshou/data/collector.py

```diff
@@ -109,12 +110,14 @@ def __init__(self,
         self.reset()
 
     @staticmethod
-    def _default_rew_metric(x):
+    def _default_rew_metric(
+        x: Union[Number, np.number]
+    ) -> Union[Number, np.number]:
         # this internal function is designed for single-agent RL
         # for multi-agent RL, a reward_metric must be provided
         assert np.asanyarray(x).size == 1, \
-            'Please specify the reward_metric ' \
-            'since the reward is not a scalar.'
+            "Please specify the reward_metric " \
```

Comment:
I would prefer never use `\` but rather `(...)`:
```
assert np.asanyarray(x).size == 1, (
    "Please specify the reward_metric "
    "since the reward is not a scalar.")
```

Reply:
Agree. This is what `black` preferred.